### DISCUSSION TITLE
Issue with sending PATCH to a collection

### DISCUSSION THREAD

**Discussion by rithu1998** [directus/directus#5341] [Archive]
Hi,
I am using Directus9 v9.0.0-rc60 and I am not able to send a PATCH to one of my collections. Currently I am getting a "503 service unavailable" message every time I try to send a patch to that certain collection. All of the other collections are able to receive a PATCH and are updated successfully within Directus. Here is a screenshot of the result within Postman.
 
<img width="965" alt="Screen Shot 2021-04-28 at 3 18 54 PM" src="https://user-images.githubusercontent.com/27563697/116479866-358b8080-a835-11eb-997f-48d93fc76dab.png">

**Comment by rijkvanzanten** [comment]
What collection? What does your schema look like? Do you have any hooks/endpoints other extensions?

**Reply by rithu1998** [reply]
The collection is named course_classification_variant and the schema has 10 collections in total and 3 of them are collections storing the many to many relationships. We have an items.create webhook for each collection and an items.update.before event hook in the /extensions/hooks directory.

**Reply by rijkvanzanten** [reply]
Try removing the hooks for a second, just to rule out your custom code isn't causing any havoc 🙂

**Reply by rithu1998** [reply]
Guess it was wreaking havoc, sorry for the unnecessary trouble and thanks for the suggestion!
